prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -224,6 +230,8 @@ void rxr_pkt_handle_ctrl_sent(struct rxr_ep *rxr_ep, struct rxr_pkt_entry *pkt_e
 		break;
 	case RXR_MEDIUM_MSGRTM_PKT:
 	case RXR_MEDIUM_TAGRTM_PKT:
+	case RXR_DC_MEDIUM_MSGRTM_PKT:
+	case RXR_DC_MEDIUM_TAGRTM_PKT:
```

Comment:
I think for DC medium, we do not want to do anything either, so these two should be moved down to be together with ```DC_EAGER_MSGRTM``` etc.

Reply:
We need to increment `tx_entry->bytes_sent` (not `tx_entry->bytes_acked`) in `rxr_pkt_handle_ctrl_sent` for `RXR_DC_MEDIUM_MSGRTM_PKT` and `RXR_DC_MEDIUM_TAGRTM_PKT`